include/cassandra.h

```diff
@@ -20,7 +20,7 @@
 #include <stddef.h>
 
 #if !defined(CASS_STATIC)
-#  if defined(_WIN32)
+#  if defined(_MSC_VER)
```

Comment:
Same from PR 161:

In the event MinGW is used to compile the source the symbol will not be exported to the DLL. Was that the intention?